src/App.js

```diff
@@ -34,14 +34,14 @@ function App() {
 
   // TODO: Alex L - try to redo the logic using todoApi
   const completeTodo = index => {
-    const newTodos = [...todos];
+    const newTodos = [...todoApi.getAll()];
```

Comment:
Not a bad idea, but not what I was looking for. You need to actually update the service
```js
const storedTodos = [...todoApi.getAll()];
// Get todo that you will update
const todoToUpdate = storedTodos[index];
// call api to update todo
todoApi.updateByID(todo.todoToUpdate, { isCompleted: true })
// Get the latest updated todos after update
setTodos([...todoApi.getAll()]);
```